Assets/MRTK/StandardAssets/Shaders/MRTK.Shaders.sentinel

```diff
@@ -1 +1 @@
-ver: 2
\ No newline at end of file
+ver: 2
```

Comment:
This looks like the version in the sentinel file was not modified.... It needs to be for the tooling to properly notify customers of changes.

Reply:
We decided that since the sentinel has already been incremented in an unreleased PR, we didn't need to increment it again here

https://github.com/microsoft/MixedRealityToolkit-Unity/pull/10096#discussion_r673472062